pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -398,6 +395,12 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 		"podAnnotations":     podAnnotations,
 	}
 
+	if svc.Config.Data["imageRegistry"] != "" {
```

Comment:
Maybe we should fall back to the current hardcoded value, if the value in data is empty?

Reply:
I hardcoded it into the component.